packages/manager/apps/dedicated/client/app/config/constants.config.js

```diff
@@ -595,6 +595,11 @@ const constants = {
       },
       pcc: {
         fr_FR: [
+          {
+            title: 'core_pcc_top_guide_11_title',
+            atInternetClickTag: 'TopGuide-PrivateCloud-11',
+            url: 'https://docs.ovh.com/fr/nutanix/',
+          },
```

Comment:
**nitpick:** Could you please follow the order. I see 11 in the title but placed above guide 1.

Reply:
hi @anooparveti , yes because we want to display title 11 on the top of others. If I am going to change the order then I need to renumber every title and see where all we are using it. That's why I kept title 11 on top of title 1.